x-pack/plugin/text-structure/qa/text-structure-with-security/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'elasticsearch.yaml-rest-test'
+
+dependencies {
+  yamlRestTestImplementation project(path: xpackModule('core'))
+  yamlRestTestImplementation project(path: xpackModule('core'), configuration: 'testArtifacts')
+  yamlRestTestImplementation project(path: xpackProject('plugin').path, configuration: 'testArtifacts')
+}
+
+// bring in text structure rest test suite
+restResources {
+  restApi {
+    includeCore '_common', 'cluster', 'nodes', 'indices', 'index', 'search', 'get', 'count', 'ingest', 'bulk'
```

Comment:
We probably don't need all these APIs to test text structure.  I guess the list is copied from the ML tests.  Maybe just `_common` is required here.

Reply:
I will see whats up.